### PR TITLE
added: migration support multiple dir

### PR DESCRIPTION
**Pull request by cq-z**
Hello!

* Type: new feature
* Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines][:contrib:]
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR

Small description of change:

added: migration support multiple dir

[:contrib:]: https://github.com/phalcon/phalcon-devtools/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: scripts/Phalcon/Version/IncrementalItem.php [hunk 1]
```diff
@@ -215,6 +221,24 @@ public function getVersion()
     {
         return $this->version;
     }
+    /**
+     * Get integer payload of the Path
```
Comment (sergeyklay) [inline scripts/Phalcon/Version/IncrementalItem.php]: "integer payload"  ? I have no idea what is it

File: scripts/Phalcon/Commands/Builtin/Migration.php [hunk 2]
```diff
@@ -43,21 +43,21 @@ class Migration extends Command
...
-            'descr=s'           => 'Migration description (used for timestamp based migration)',
-            'data=s'            => 'Export data [always|oncreate] (Import data when run migration)',
-            'force'             => 'Forces to overwrite existing migrations',
-            'ts-based'          => 'Timestamp based migration version',
-            'log-in-db'         => 'Keep migrations log in the database table rather than in file',
-            'dry'               => 'Attempt requested operation without making changes to system (Generating only)',
-            'verbose'           => 'Output of debugging information during operation (Running only)',
+            'action=s' => 'Generates a Migration [generate|run]',
+            'config=s' => 'Configuration file',
+            'migrations=s' => 'Migrations directory,multiple dir use `,` separate',
```
Comment (sergeyklay) [inline scripts/Phalcon/Commands/Builtin/Migration.php]:
```diff
- Migrations directory,multiple dir use `,` separate
+ Migrations directory. Use comma separated string to specify multiple directories
```

File: scripts/Phalcon/Version/IncrementalItem.php (hunk 1 above)
Comment (sergeyklay) [reply scripts/Phalcon/Version/IncrementalItem.php]: Could you rephrase this

Comment (cq-z) [reply scripts/Phalcon/Version/IncrementalItem.php]: Is it correct now?

**Comment by sergeyklay** [comment]
Thank you
